### PR TITLE
feat: showcase layout

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: apps/showcase/src/styles.scss [hunk 1]
```diff
@@ -1,28 +1,31 @@
-// Issue with pnp
-// @import "bootstrap/scss/bootstrap";
-// @import "bootstrap-icons/font/bootstrap-icons";
-@import "https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css";
-@import "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css";
+// FIXME sass-loader cannot resolve scss files with pnp
```
Comment (kpanot) [inline apps/showcase/src/styles.scss:1]: seems not required anymore

File: apps/showcase/src/app/app.component.html [hunk 2]
```diff
@@ -1,11 +1,38 @@
...
-      [linksGroups]="linksGroups"
-      [activeUrl]="activeUrl$ | async"
-    ></o3r-sidenav-pres>
+<header>
+  <div class="header-icon">
+    <img src="assets/otter-sticker.svg" width="48" height="48" alt="Otter logo" />
   </div>
-  <div class="page-content flex-fill">
-    <router-outlet></router-outlet>
+  Otter Showcase
```
Comment (kpanot) [inline apps/showcase/src/app/app.component.html:5]: missing localization

File: apps/showcase/src/app/app.component.html [hunk 3]
```diff
@@ -1,11 +1,38 @@
...
+    </div>
   </div>
 </div>
+<footer>
+  <div class="container">
+    <div class="row">
+      <div class="col-8">
+      </div>
+      <div class="col-4">
+        <h5>Links</h5>
```
Comment (kpanot) [inline apps/showcase/src/app/app.component.html:26]: missing loc

File: apps/showcase/src/assets/otter-sticker.svg
Comment (kpanot) [inline apps/showcase/src/assets/otter-sticker.svg:1]: the left hear halo is a bit strange

File: apps/showcase/src/app/app.component.scss [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
+  top: 0;
+  height: 4rem;
+  width: 100%;
+  background: var(--bs-primary);
+  box-shadow: 0 0.5rem 1rem rgba(0,0,0,0.15), inset 0 -1px 0 rgba(255,255,255,0.15);
+  z-index: 1021;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding-left: 1rem;
```
Comment (kpanot) [inline apps/showcase/src/app/app.component.scss:12]: should not be handle by bootstrap css class (same for margins)?

File: apps/showcase/src/app/app.component.html (hunk 3 above)
Comment (cpaulve-1A) [reply apps/showcase/src/app/app.component.html:26]: do we really use the translation reco from otter in the showcase?

File: apps/showcase/src/styles.scss (hunk 1 above)
Comment (fpaul-1A) [reply apps/showcase/src/styles.scss:1]: we can import `css` files but not `scss` files

File: apps/showcase/src/assets/otter-sticker.svg
Comment (fpaul-1A) [reply apps/showcase/src/assets/otter-sticker.svg:1]: I've just changed the color :D
anyway the icon is displayed as 48x48 so I don't think it's very noticeable

File: apps/showcase/src/app/app.component.html (hunk 3 above)
Comment (kpanot) [reply apps/showcase/src/app/app.component.html:26]: If the "showcase" is a showcase of the Otter capabilities, I would say yes :)

Comment (fpaul-1A) [reply apps/showcase/src/app/app.component.html:26]: it needs to be done globally on all the texts of the showcase app, it should be done in a separate PR

File: apps/showcase/src/app/app.component.scss (hunk 4 above)
Comment (fpaul-1A) [reply apps/showcase/src/app/app.component.scss]: removed

File: apps/showcase/src/app/app.component.html (hunk 2 above)
Comment (fpaul-1A) [reply apps/showcase/src/app/app.component.html]: localization will be handled separately

**Comment by fpaul-1A** [comment]
> Could be good to have some screenshot :)

image

**Pull request merged**